### PR TITLE
add customKeyIdentifier to password credential

### PR DESCRIPTION
**Pull request by yugangw-msft**
//CC: @lmazuel. CLI users want this

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.

### CODE REVIEW

**Comment by yugangw-msft** [comment]
@lmazuel, please review. If you are happy, please publish the python sdk

File: specification/graphrbac/data-plane/stable/1.6/graphrbac.json [hunk 1]
```diff
@@ -2465,7 +2465,6 @@
         },
         "customKeyIdentifier": {
           "type":"string",
-          "format": "byte",
```
Comment (yugangw-msft) [inline specification/graphrbac/data-plane/stable/1.6/graphrbac.json]: Cert based id should be a plain string with the thumbprint as the value

Comment (lmazuel) [reply specification/graphrbac/data-plane/stable/1.6/graphrbac.json]: > byte – base64-encoded characters, for example, U3dhZ2dlciByb2Nrcw==

So it depends only if you have base64 string on the wire. If yes, the right value is "byte". If no, the right value is "string"

File: specification/graphrbac/data-plane/stable/1.6/graphrbac.json [hunk 2]
```diff
@@ -2494,6 +2493,11 @@
         "value": {
           "type": "string",
           "description": "Key value."
+        },
+        "customKeyIdentifier": {
+          "type": "string",
+          "format": "byte",
```
[the remark this answers is not in the document]
Comment (lmazuel) [reply specification/graphrbac/data-plane/stable/1.6/graphrbac.json]: Checked on Fiddler, this one is definitley base64 (and then `byte`)

File: specification/graphrbac/data-plane/stable/1.6/graphrbac.json (hunk 1 above)
Comment (lmazuel) [reply specification/graphrbac/data-plane/stable/1.6/graphrbac.json]: After check, this string is actually not byte on the wire and the fix is right.

**Review by lmazuel — APPROVED** [review]
Double checked and tested with CLI

**Pull request merged**